src/server/routers/app.ts

```diff
@@ -0,0 +1,14 @@
+import * as trpcNext from '@trpc/server/adapters/next';
+import { router } from '../trpc';
+import { reviewsRouter } from './review';
+
+export const appRouter = router({
+  reviews: reviewsRouter,
+});
```

Comment:
I expect more routers in the future so this is the basic setup for merging routers